controllers/vgmanager_test.go

```diff
@@ -0,0 +1,85 @@
+package controllers
+
+import (
+	"context"
+	"testing"
+
+	lvmv1alpha1 "github.com/red-hat-storage/lvm-operator/api/v1alpha1"
+	"gotest.tools/v3/assert"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	testNamespace = "default"
+)
+
+func newFakeLVMClusterReconciler(t *testing.T, objs ...client.Object) *LVMClusterReconciler {
+	scheme, err := lvmv1alpha1.SchemeBuilder.Build()
+	assert.NilError(t, err, "creating scheme")
+
+	err = corev1.AddToScheme(scheme)
+	assert.NilError(t, err, "adding corev1 to scheme")
+
+	err = appsv1.AddToScheme(scheme)
+	assert.NilError(t, err, "adding appsv1 to scheme")
+
+	err = storagev1.AddToScheme(scheme)
+	assert.NilError(t, err, "adding appsv1 to scheme")
```

Comment:
Log is incorrect. It should be "adding storagev1 to scheme"

Reply:
fixed.